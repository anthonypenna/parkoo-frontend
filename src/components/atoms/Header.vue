<template>
  <header class="header">
    <h1 class="header__title">parkoo</h1>
  </header>
</template>

<style lang="scss" scoped>
.header {
  position: fixed;
  left: 0;
  top: 0;
  width: 100%;
  background: #fff;
  border-bottom-left-radius: 15px;
  border-bottom-right-radius: 15px;
  box-shadow: 0 4px 16px rgba($color: #000000, $alpha: 0.1);
  z-index: 10;

  &__title {
    margin: 0;
    padding: 8px 0;
    font-size: 32px;
    font-weight: normal;
    text-align: center;
  }
}
</style>
