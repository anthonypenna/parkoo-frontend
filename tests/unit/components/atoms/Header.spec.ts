import { shallowMount } from '@vue/test-utils'
import Header from '@/components/atoms/Header.vue'

describe('<Header />', () => {
  it('should render correctly', () => {
    const wrapper = shallowMount(Header)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
