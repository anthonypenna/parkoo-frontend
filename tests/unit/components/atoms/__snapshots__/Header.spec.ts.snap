// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<header class="header">
  <h1 class="header__title">parkoo</h1>
</header>
`;
